server/src/resolvers/UserResolver.ts

```diff
@@ -29,4 +31,18 @@ export class UserResolver {
 			};
 		}
 	}
+	@Mutation(() => UserResponse, {
+        description: "Updates user.",
+    })
+    async updateUser(
+        @Arg("data", () => UpdateUserType) data: UpdateUserType) {
+		const errors: FieldError[] = [];
```

Comment:
Create a function in validate.ts that does all of this, then just call it with something like validateUpdate(data).
also you don't need the ! when calling data.something since it will cause errors later on if the users doesn't give that info, just use an if statement to check if it exists.
`if(data.something) {
  const errXXX = validateXXX(data.XXX);
  if (errXXX) errors.push(errXXX);
}`